web/styles/message_row.css

```diff
@@ -776,6 +776,11 @@ of the base style defined for a read-only textarea in dark mode. */
         margin-left: 0;
         margin-top: 0;
     }
+
+    .preview_message_area {
+        /* Share the same max-height as .message_edit_content */
+        max-height: 24em;
```

Comment:
This comment smells like a CSS variable we should create.

Reply:
Yes it does. I'll push a fix.